syntax = "proto3";

service OrderService {
    rpc PlaceOrder (OrderRequest) returns (OrderResponse);
}

message OrderRequest {
    string product_name = 1;
    float price = 2;
    string payment_gateway = 3;
    string card_brand = 4;
    string bank = 5;
    string shipping_address = 6;
    string region = 7;
    string email = 8;
}

message OrderResponse {
    string status = 1;
    string order_id = 2;
}
